js/index.js

```diff
@@ -0,0 +1,98 @@
+import lozad from "lozad";
+import checkElement from "../utils/dom";
+
+import Menu from "./models/menu";
+import * as menuView from "./views/menuView";
+
+import * as navMenus from "../utils/navMenus";
+
+const state = {};
+
+function addDisplayOnHover() {
+  const elementContained = document.querySelectorAll(".article");
+  let clickedElement;
+  Array.from(elementContained).forEach(async elem => {
```

Comment:
A little trick to transform a list of nodes to an array is using the spread operator like this
`[...listOfNodes].map`

Reply:
Yeah thanks, actually I learned it because I apply it with how i implement the lazy loading, I forgot to change all the other ones.